console.cpp

```diff
@@ -0,0 +1,47 @@
+#include <Windows.h>
+#include <tchar.h>
+#include "console.hpp"
+#include "cpp_tstring.hpp"
+
+_console::_console() {
+	this->output = GetStdHandle(STD_OUTPUT_HANDLE);
+	this->input = GetStdHandle(STD_INPUT_HANDLE);
+	this->error = GetStdHandle(STD_ERROR_HANDLE);
+}
+
+void _console::writeOutput(const TCHAR str[]) {
```

Comment:
consider using std::string_view for this

Reply:
For writing output? (const tchar[] to a string view?)

Reply:
You mean adding an overload of console functions that accept string views instead?

Reply:
Add both. Also add overloads for `tstring`, `tstring const&`, and `tstring &&` for good measure. Maybe there can be a C++20 concept for any string-like type and use metaprogramming to convert/move when needed. For anyone else with better ideas for passing strings to functions, feel free to chime in.

Reply:
const tstring is automatically compatible with normal tstring, and there is no semantic difference between const tstring& and tstring const&; the language treats them as the same type. I added one overload for passing the string as an rvalue reference: commit 950127b69cd51995ec213f2b2dafd3f851582ca2